### PR TITLE
refactor: add dusk selector to 2fa form

### PR DESCRIPTION
**Pull request by Highjhacker**
## Summary

Related to https://app.clickup.com/t/hk5vqv

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: resources/views/profile/two-factor-authentication-form.blade.php [hunk 1]
```diff
@@ -142,69 +144,76 @@ class="w-full button-secondary sm:w-auto"
...
-                <div class="flex flex-col-reverse w-full sm:flex-row sm:justify-between">
-                    <div class="flex justify-center w-full mt-3 sm:justify-start sm:mt-0">
-                        <x-ark-file-download
-                            :filename="'2fa_recovery_code_' . $this->user->name"
-                            :content="implode('\n', json_decode(decrypt($this->user->two_factor_recovery_codes)))"
-                            :title="trans('fortify::actions.download')"
-                            wrapper-class="w-full sm:w-auto"
-                            class="justify-center w-full"
-                        />
+    <div dusk="recovery-codes-modal">
```
Comment (leMaur) [inline resources/views/profile/two-factor-authentication-form.blade.php:159]: @Highjhacker Is there a reason why you put the `<div>` outside the `@if($this->modalShown)`?
I think it'll be better inside the `@if` so it won't show in the DOM all the time. Or may cause issues with Dusk?

Comment (Highjhacker) [reply resources/views/profile/two-factor-authentication-form.blade.php:159]: Yeah was related to dusk, didn't really liked it either but couldn't make it work properly without it 🤔

**Review by leMaur — APPROVED** [review]

File: resources/views/profile/two-factor-authentication-form.blade.php [hunk 2]
```diff
@@ -142,69 +144,76 @@ class="w-full button-secondary sm:w-auto"
                     @lang('fortify::actions.recovery_codes')
                 </button>
 
-                <button type="submit" class="w-full button-primary sm:w-auto" wire:click="disableTwoFactorAuthentication">
+                <button
+                    type="submit"
+                    class="w-full button-primary sm:w-auto"
+                    wire:click="disableTwoFactorAuthentication"
+                    dusk="disable-one-time-password"
```
Comment (ItsANameToo) [inline resources/views/profile/two-factor-authentication-form.blade.php]:
```suggestion
                    dusk="disable-two-factor-authentication"
```

OTP is the password itself, you enable and disable 2FA with it.

File: resources/views/profile/two-factor-authentication-form.blade.php [hunk 3]
```diff
@@ -111,6 +112,7 @@ class="hide-number-input-arrows"
                     type="button"
                     class="w-full button-secondary sm:w-auto"
                     wire:click="enableTwoFactorAuthentication"
+                    dusk="enable-one-time-password"
```
Comment (ItsANameToo) [inline resources/views/profile/two-factor-authentication-form.blade.php]:
```suggestion
                    dusk="enable-two-factor-authentication"
```

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**